Source/Models/FORMTarget.h

```diff
@@ -18,20 +19,22 @@ typedef NS_ENUM(NSInteger, FORMTargetActionType) {
     FORMTargetActionNone
 };
 
-@interface FORMTarget : NSObject
+@interface FORMTarget : FORMFieldElement
 
 @property (nonatomic, copy) NSString *targetID;
-@property (nonatomic, copy) id targetValue;
+@property (nonatomic, copy) id value;
 @property (nonatomic, copy) NSString *typeString;
 @property (nonatomic, copy) NSString *actionTypeString;
 @property (nonatomic, copy) NSString *condition;
 
-@property (nonatomic) FORMFieldValue *value;
+@property (nonatomic) FORMFieldValue *fieldValue;
```

Comment:
@3lvis There were the following changes:

 `targetValue` -> `value`
 `value` -> `fieldValue`

So I can't make `fieldValue` point to `value`. Maybe we could just back to previous names not to break backwards compatibility?